.travis.yml

```diff
@@ -125,6 +127,10 @@ before_install:
       pushd $HOME/Library/Python/$PYMVER*/bin
       export PATH="$PWD:$PATH"
       popd
+      # fix CMake variables
+      export BUILD_FLAGS="$BUILD_FLAGS -DPYTHON_LIBRARY=$(python-config --prefix)/lib/libpython2.7.dylib"
```

Comment:
what happens if you just don't set `PYTHON_LIBRARY` etc, only `PYTHON_EXECUTABLE`?